### PR TITLE
Add get sessions by event

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: Gateway/app.config [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <add key="LastContactUrl" value="http://comms:8080/last-contact/"/>
     <add key="EventsUrl" value="http://sessions:8080/events/"/>
     <add key="SessionsUrl" value="http://sessions:8080/sessions/"/>
-    <add key="SessionIdsUrl" value="http://sessions:8080/sessionIds/"/>
+    <add key="SessionIdsUrl" value="http://sessions:8080/session/ids/"/>
```
Comment (pallenSL) [inline Gateway/app.config]: Why isn't sessions plural?

The API isn't the place for creativity.  If we don't follow a pattern we need to create, update, and read documents.

**Comment by niksoper** [comment]
I think this looks good for the current API design but I've spoken to @jebbinSCL and we think it would be better to move to the following to get session IDs for a given event:

`/events/{eventId}/sessions/ids`

The thought behind this is we are conceptually getting a specific event and then retrieving its session IDs. With the current approach of `sessions/ids?eventId=...` the implication is that the query parameter is optional and leaving it out will get all session IDs, which is not the case.

I wanted to add a task to Jira to update the routes but I haven't got access at the moment - IT ticket raised.

**Comment by niksoper** [comment]
so :+1:  for this work :smile:

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**